sidecar/src/service/tracing/trace_flusher.rs

```diff
@@ -78,16 +82,26 @@ impl TraceFlusherMetrics {
 /// `TraceFlusher` is a structure that manages the flushing of traces.
 /// It contains the traces to be sent, the flusher task, the interval for flushing,
 /// the minimum sizes for force flushing and dropping, and the remote configs.
-#[derive(Default)]
 pub(crate) struct TraceFlusher {
     inner: Mutex<TraceFlusherData>,
-    pub(crate) interval: AtomicU64,
-    pub(crate) min_force_flush_size: AtomicU32,
-    pub(crate) min_force_drop_size: AtomicU32, // put a limit on memory usage
+    pub(crate) interval_ms: AtomicU64,
+    pub(crate) min_force_flush_size_bytes: AtomicU32,
+    pub(crate) min_force_drop_size_bytes: AtomicU32, // put a limit on memory usage
     remote_config: Mutex<AgentRemoteConfigs>,
     pub metrics: Mutex<TraceFlusherMetrics>,
 }
-
+impl Default for TraceFlusher {
```

Comment:
Previously, everything defaulted to 0 which made it unusable. There is no point in having defaults if you can't use them. These values are are a best guess from existing tests. In reality, the sidecar-ffi code is setting the values based on the config, so this shouldn't impact existing usage.